app/uk/gov/hmrc/cataloguefrontend/util/MarkdownLoader.scala

```diff
@@ -34,4 +35,14 @@ object MarkdownLoader {
       finally source.close()
     transformer.transform(lines.take(maxLines).mkString("\n")).getOrElse("<Unable to render at this time>")
   }
+
+  def markdownFromString(s: String): Either[DocumentParser.ParserError, String] = {
+    val transformer = Transformer
```

Comment:
We can delete this transformer and reuse the one already available in `MarkdownLoader` (can it be made private?)
.
We could rename `loadAndRenderMarkdownFile` to `markdownFromFile` for consistency.
A consistent return type would be good too - I'm thinking `Either[String, String]` where the left value is a simple error message (could even be the `Unable to render markdown` message) - so we can keep the markdown library encapsulated, but let clients decide how they want to treat it.